app/src/main/java/io/pslab/fragment/LALogicLinesFragment.java

```diff
@@ -1188,13 +1188,13 @@ protected Void doInBackground(ArrayList<String>... arrayLists) {
                 scienceLab.startFourChannelLA(null, null, modes, null, triggerChannel);
                 delayThread(1000);
                 LinkedHashMap<String, Integer> data = scienceLab.getLAInitialStates();
-                delayThread(500);
+                delayThread(1000);
                 holder1 = scienceLab.fetchLAChannel(channelNumber1, data);
-                delayThread(500);
+                delayThread(1000);
                 holder2 = scienceLab.fetchLAChannel(channelNumber2, data);
-                delayThread(500);
+                delayThread(1000);
                 holder3 = scienceLab.fetchLAChannel(channelNumber3, data);
-                delayThread(500);
+                delayThread(1000);
```

Comment:
Are these delays (1000 from 500) necessary?

Reply:
I checked with 500 .... Sometimes it fails to fetch data. I guess it is safe to keep 1000.